apps/s_server.c

```diff
@@ -2777,8 +2777,12 @@ static void print_connection_info(SSL *con)
 #endif
     if (SSL_session_reused(con))
         BIO_printf(bio_s_out, "Reused session-id\n");
-    BIO_printf(bio_s_out, "Secure Renegotiation IS%s supported\n",
-               SSL_get_secure_renegotiation_support(con) ? "" : " NOT");
+    if ((SSL_get_options(con) & SSL_OP_NO_RENEGOTIATION))
+        BIO_printf(bio_s_out, "Renegotiation is DISABLED\n");
+    else
+        BIO_printf(bio_s_out, "Secure Renegotiation IS%s supported\n",
+                   SSL_get_secure_renegotiation_support(con) ? "" : " NOT");
```

Comment:
@mattcaswell had a point that we still want to indicate that secure renegotiation is supported, so that clients will not abort the connection. So, I think I may redo this.